src/index/vocabulary/PolymorphicVocabulary.h

```diff
@@ -159,6 +159,21 @@ class PolymorphicVocabulary {
         vocab_);
   };
 
+  // Check if `getGeoInfo` can return results because there is a
+  // `GeoVocabulary`.
```

Comment:
This comment is a little different for each vocabulary, 
please unify (I like the one from splitVocabulary)